ch_backup/clickhouse/disks.py

```diff
@@ -142,9 +143,23 @@ def _create_temporary_disk(
         disk_config["endpoint"] = os.path.join(
             f"{endpoint.scheme}://{endpoint_netloc}", source_bucket, source_path, ""
         )
+        disks_config = {tmp_disk_name: disk_config}
+
+        request_timeout_ms = int(disk_config.get("request_timeout_ms", 0))
+        if request_timeout_ms < CH_OBJECT_STORAGE_REQUEST_TIMEOUT_MS:
+            disks_config[tmp_disk_name]["request_timeout_ms"] = str(
+                CH_OBJECT_STORAGE_REQUEST_TIMEOUT_MS
+            )
+            disks_config[disk_name] = {
+                "request_timeout_ms": {
+                    "@replace": "replace",
+                    "#text": str(CH_OBJECT_STORAGE_REQUEST_TIMEOUT_MS),
+                }
+            }
+
```

Comment:
have tested manualy and:
```
root@clickhouse01:/# cat /tmp/clickhouse-disks-config.xml 
<?xml version="1.0" encoding="utf-8"?>
<yandex>
        <storage_configuration>
                <disks>
                        <hdd1>
                                <path>/hdd1/</path>
                        </hdd1>
                        <hdd2>
                                <path>/hdd2/</path>
                        </hdd2>
                        <s3>
                                <type>s3</type>
                                <endpoint>http://minio01:9000/cloud-storage-01/data/</endpoint>
                                <access_key_id>***</access_key_id>
                                <secret_access_key>**</secret_access_key>
                        </s3>
                        <s3_source>
                                <type>s3</type>
                                <endpoint>http://minio01:9000/cloud-storage-01/data/</endpoint>
                                <access_key_id>**</access_key_id>
                                <secret_access_key>**</secret_access_key>
                        </s3_source>
                </disks>
        </storage_configuration>
```
Is that exepected? 

Ps. Im not sure if the test runned correctly due to broken CI, will recheck after green CI.

Reply:
Can't understand. ch-backup doesn't modify original config files, it creates new one with name `cloud_storage_tmp_disk_<disk_name>_source.xml`. So `clickhouse-disks-config.xml` - it's an original file, not created by ch-backup.

Reply:
`/tmp/clickhouse-disks-config.xml` not `/var/lib/clcikhouse/...` We are generating additional config for `clickhouse-disks` usage.

Reply:
Fixed